docs/builtins.rst

```diff
@@ -0,0 +1,296 @@
+=================
+Functions in L20n
+=================
+
+Functions provide additional functionality available to the localizers.
+They can be either used to format data according to the current language's
+rules or can provide additional data that the localizer may use (like, the
+platform, or time of the day) to fine tune the translation.
+
+L20n ships with a number of built-in functions that can be used from within
+localization messages.
+
+The list of available functions is extensible and environments may want to
+add additional functions designated to aid localizers writing translations
+targeted for a given environment.
+
+
+How to use Functions
+====================
+
+FTL Functions can only be called inside of placeables. Use them to return a
+value to be interpolated in the message or as selectors in select expressions.
+
+Example::
+
+
+  today-is = Today is { DATETIME($date) }
+
+
+Function parameters
+===================
+
+Functions may (but don't have to) accept positional and keyword arguments.
+Some parameters are only available to developers passing the Function as
```

Comment:
Maybe: `Some keyword parameters are only available to developers and cannot be used inside of FTL translations.  Instead they can be set on external arguments passed into translations by the developer.`